hipe4ml/model_handler.py

```diff
@@ -358,6 +358,11 @@ def optimize_params_optuna(self, data, hyperparams_ranges, cross_val_scoring, nf
             A string indicating the filename of the study. If None,
             the study is not saved into a file.
 
+        early_stopping: int
```

Comment:
```suggestion
        early_stopping_lim: float
```
It can't be an int, and a more meaningful name is better (not sure my proposal is the right one).